modules/node_modules/@ncigdc/uikit/Modal.js

```diff
@@ -21,7 +21,7 @@ const modalStyles = {
     background: 'rgb(255, 255, 255)',
     borderRadius: '4px',
     margin: '30px auto',
-    padding: '15px',
+    padding: '0px',
```

Comment:
There are atleast two more in `containers/CaseAggregations.js` and `containers/FileAggregation.js`